server/public/model/outgoing_oauth_connection.go

```diff
@@ -57,6 +57,34 @@ func (oa *OutgoingOAuthConnection) Sanitize() {
 	oa.CredentialsPassword = nil
 }
 
+// Patch updates the OutgoingOAuthConnection object with the non-empty fields from the given connection.
+func (oa *OutgoingOAuthConnection) Patch(conn *OutgoingOAuthConnection) {
+	if conn.Name != "" {
```

Comment:
Fine but noting the assumption of `conn` always being non-nil.

Reply:
Added a condition to end early if a connection is not provided.